apparmor.d/profiles-a-f/firewalld

```diff
@@ -50,10 +52,11 @@ profile firewalld @{exec_path} {
   @{bin}/false                    rix,
   @{bin}/ipset                    rix,
   @{bin}/kmod                     rPx,
+  @{bin}/modprobe                 rPx,
```

Comment:
Because of the `no new privs` flag, this need to be `rix` (that's why you get the content of the profile raised as log in the firewalld profile).

Reply:
Just `rix,` or redirect to `-> kmod` as in `aa-log -r`?

Reply:
just `rix`, because `-> kmod` [won't work](https://apparmor.pujol.io/development/structure/#no-new-privileges).